devtools/conda_install_reqs.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/bash
+
+# USAGE
+# ./conda_install_reqs.sh
+#   Install the requirements. If the environment variable OPS_ENV is set,
+#   then create a new environment called $OPS_ENV. Note that you MUST set
+#   the environment variable CONDA_PY (the version of Python to use, e.g.,
```

Comment:
This (`CONDA_PY`) is an extra requirement for downstream testing, could you build-in a fallback like 
```
CONDA_PY=`python --version | cut -d' ' -f2`
```
if `CONDA_PY` is not defined?

Reply:
Actual code is a bit more complicated, because (1) `python --version` goes to stderr, so have to redirect; (2) I only want to restrict to major/minor; don't care about patch-level. But this is done.